src/components/shared/AlertBox.tsx

```diff
@@ -109,7 +109,7 @@ const AlertBox = forwardRef<any, Props>(function NavLinkRef(
                           ...SHARED_STYLING,
                           borderLeftColor:
                               theme.palette[severity][theme.palette.mode],
-                          color: theme.palette[severity][theme.palette.mode],
+                          color: alertTextPrimary[theme.palette.mode],
```

Comment:
With icon noir the icons are not readable with some colors so making them match the text